src/components/App.jsx

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
```

Comment:
### Optional
- [ ] It would be better if we improve on our commit message going forward, for example; write `Add` instead of `Added`, `Remove` instead of `Removed`, `Fix` instead of `Fixed`, `Correct` instead of `Corrected`, etc, and let us try to avoid typos